src/Slices/Status/UI/StatusItem.tsx

```diff
@@ -52,10 +71,43 @@ const Category = styled.span`
 
 const InlineTitle = styled(Title)`
   display: inline-block;
-  padding-right: 8px;
+  padding-right: var(--pf-v5-global--spacer--sm);
 `;
 
 const CompactDescriptionList = styled(DescriptionList)`
   --pf-v5-c-description-list--m-compact--RowGap: 0;
-  margin-bottom: 16px;
+  margin-bottom: var(--pf-v5-global--spacer--md);
 `;
+
+const IndentedDescriptionListGroup = styled(DescriptionListGroup)`
+  padding-left: var(--pf-v5-global--spacer--md);
+`;
+
+interface SubListProps {
+  name: string;
+  properties: Record<string, string>;
+}
+
+/**
+ * Renders sub list for Status value that is a Record instead of string.
+ *
+ * @props {SubListProps} props - The properties for the SubList component.
+ * @prop {string} name - The name of the property.
+ * @prop {Record<string, string>} properties - The sub properties to display in the SubList components.
+ * @returns {React.FC<SubListProps>} The rendered SubList component.
+ */
+const SubList: React.FC<SubListProps> = ({ name, properties }) => {
```

Comment:
NestedListItem would be better, we have the StatusList component, so it makes it a bit confusing to name this one also a list, when it's actually an item.